package.json

```diff
@@ -12,6 +12,7 @@
   },
   "scripts": {
     "dev": "webpack-dev-server --env.env=dev",
+    "start": "yarn dev",
```

Comment:
Unrelated, but I keep getting mixed up between `yarn start` and `yarn dev` as different projects use different ones to start the dev app. So let's just allow both! 😄